.github/workflows/free-disk-space/action.yml

```diff
@@ -5,4 +5,31 @@ runs:
   steps:
       - name: Remove /usr/local/* to free disk space
         run:  sudo rm -rf /usr/local/*
-        shell: bash
+        shell: /usr/bin/bash -e {0}
+      - name: Remove /opt/* to free disk space
+        run:  sudo rm -rf /opt/*
+        shell: /usr/bin/bash -e {0}
+      - name: Remove .net to free disk space
+        shell: /usr/bin/bash -e {0}
+        run:  sudo apt purge dotnet*
+      - name: Remove chrome to free disk space
+        run:  sudo apt purge google-chrome-stable
+        shell: /usr/bin/bash -e {0}
+      - name: Remove temurin-jdk to free disk space
+        run:  sudo apt purge temurin*
+        shell: /usr/bin/bash -e {0}
+      - name: Remove google-cloud-sdk to free disk space
+        run:  sudo apt purge google-cloud-sdk
+        shell: /usr/bin/bash -e {0}
+      - name: Remove hhvm to free disk space
+        run:  sudo apt purge hhvm
+        shell: /usr/bin/bash -e {0}
+      - name: Remove azure-cli to free disk space
+        run:  sudo apt purge azure-cli
+        shell: /usr/bin/bash -e {0}
+      - name: Remove llvm-* to free disk space
+        run:  sudo apt purge llvm-*
+        shell: /usr/bin/bash -e {0}
+      - name: Show disk space usage
+        run:  df -h
+        shell: /usr/bin/bash -e {0}
```

Comment:
```
run: |
  sudo apt-get purge 'dotnet*' google-chrome-stable 'temurin*' google-cloud-sdk hhvm azure-cli 'mongodb*' powershell firefox chromium 'llvm*' 'libllvm*' 'adoptopenjdk*' 'mysql*' libgl1-mesa-dri apache2 'mono*'
  sudo rm -rf /usr/local/* /opt/hostedtoolcache /opt/pipx* /opt/az
```
gives us 64G

Reply:
And please drop those pointless `shell:` statements.

Reply:
Interesting links:
https://github.com/ThewApp/free-actions/blob/main/.github/workflows/free.yml
https://github.com/actions/virtual-environments/discussions/3242

Reply:
+4G if you also purge docker images.